app/src/main/res/drawable/ic_salon.xml

```diff
@@ -0,0 +1,12 @@
+<vector android:height="200dp" android:viewportHeight="382.671"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_salon` appears to be unused
 Unused resources make applications larger and slow down builds.